multimodal/src/autogluon/multimodal/data/process_text.py

```diff
@@ -243,13 +244,23 @@ def build_one_token_sequence(
             max_length = self.max_len - (len(text_tokens) + 1)
         else:
             max_length = self.max_len - 2
+        if self.eos_only:
+            # For EOS-only, the tokens will be combined as
+            # [Field1 Tokens] [EOS] [Field2 Tokens] [EOS] [Field3 Tokens] [EOS]
+            # Otherwise, the tokens will be combined as
+            # [CLS] [Field1 Tokens] [SEP] [Field2 Tokens] [SEP] [Field3 Tokens] [EOS]
+            max_length += 1
```

Comment:
Why do we need to increase the max length in the `eos_only` mode?

Reply:
The number of special tokens will be reduced by 1, so the max length we can allocate for the text field tokens will be increased by 1.

Reply:
Got it.